### PR TITLE
🐛(tasks) fix redeploying apps that have no deployment config and release 4.1.2

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

The redis-sentinel app has no dc and works only by declaring an svc. The first deployment works perfectly, but redeployments fail because we endup expecting 0 pods running when there are in fact pods running (they have been started by the k8s operator and not by a dc created
by Arnold).

## Proposal

Instead of checking if the number of running pods is equal to the number of expected pods, we can check if the number of running pods is greater than the number of expected pods.

### CODE REVIEW

File: tasks/manage_app.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@
     field_selectors:
       - "status.phase=Running"
   register: pods
-  until: pods.resources is defined and ( pods.resources | length ) == ( expected_running_pods | int )
+  until: pods.resources is defined and ( pods.resources | length ) >= ( expected_running_pods | int )
```
Comment (lunika) [inline tasks/manage_app.yml]: we could also add a `when` instruction testing if `deployments | length > 0`

```
when: deployments |  length > 0
```

Comment (jmaupetit) [reply tasks/manage_app.yml]: Indeed, I think your solution is more reliable @lunika, but why not using:

```yaml
  when: expected_running_pods | int > 0
```

**Review by jmaupetit — COMMENTED** [review]
Don't forget the CHANGELOG!

File: tasks/manage_app.yml (hunk 1 above)
Comment (sampaccoud) [reply tasks/manage_app.yml]: Ok I did this. Checking on `expected_running_pods` is easier to understand as we have this variable at hand here.

**Comment by sampaccoud** [comment]
Added the 4.1.2 fix release to this PR.

**Review by jmaupetit — APPROVED** [review]

**Comment by sampaccoud** [comment]
@jmaupetit worse GTM gif I got from you!

File: tasks/manage_app.yml (hunk 1 above)
Comment (lunika) [reply tasks/manage_app.yml]: > Indeed, I think your solution is more reliable @lunika, but why not using:
> 
> ```yaml
>   when: expected_running_pods | int > 0
> ```

Yes, you are a very clever man

**Pull request merged**